porcupine/plugins/sort.py

```diff
@@ -1,3 +1,9 @@
+"""
+Sorts the selected rows, or if there's no selection, to the next blank line.
```

Comment:
A few nits:
- Sorting happens alphabetically, which is kind of dumb in many contexts, and not necessarily what you want. That should be mentioned.
- rows --> lines. This seems to be more standard and Porcupine already shows "Line 123, column 45" in status bar.
- The logic isn't "to the next blank line". It finds the previous and next blank line, and sorts between them.

```suggestion
Sort the selected lines alphabetically, or if there's no selection, sort between surrounding blank lines.
```